### PR TITLE
Edit argo appset

### PR DESCRIPTION
**Pull request by chenz4027**
https://github.com/open-cluster-management/backlog/issues/14305

### CODE REVIEW

File: src-web/components/ArgoCreationPage/ArgoCreationPage.js [hunk 1]
```diff
@@ -207,12 +209,14 @@ class ArgoCreationPage extends React.Component {
                 : error.name
               error.name = errorName
             }
+
             const fetchControl = {
-              // resources: getApplicationResources(application),
```
Comment (jeswanke) [inline src-web/components/ArgoCreationPage/ArgoCreationPage.js:211]: delete comment?

Comment (chenz4027) [reply src-web/components/ArgoCreationPage/ArgoCreationPage.js:211]: Thanks! Will delete it

Comment (chenz4027) [reply src-web/components/ArgoCreationPage/ArgoCreationPage.js:211]: hmmm actually I turned this on

File: src-web/containers/App.js [hunk 2]
```diff
@@ -110,6 +111,10 @@ class App extends React.Component {
 
     const getSingleApplicationTabs = params => {
       const isArgoApp = _.get(location, 'search').indexOf('argoproj.io') !== -1
+      const isArgoAppset =
+        _.get(location, 'search').indexOf('&applicationset') !== -1
+      const isRemoteCluster =
+        _.get(location, 'search').indexOf('&cluster=') !== -1
```
Comment (KevinFCormier) [inline src-web/containers/App.js:117]: We should probably parse the query string here and then check for the keys/values we are interested in.

For example, searching for `&applicationset` means that applicationset cannot be the first query param, as in `?applicationset=foo`

File: nls/platform.properties [hunk 3]
```diff
@@ -99,6 +99,7 @@ argo.placement.title = Placement
 argo.template.title = Template
 argo.template.source.title = Source
 argo.template.destination.title = Destination
+argo.placement.exception = does not exist in this environment. Choose another rule or create a new one.
```
Comment (KevinFCormier) [inline nls/platform.properties:102]: Can we use a substitution here? `{0} does not exist in this...`

Comment (fxiang1) [reply nls/platform.properties:102]: I was going to suggest:
`Placement {0} does not exist in this environment. Choose another rule or create a new one.`

**Review by jeswanke — APPROVED** [review]

**Review by fxiang1 — APPROVED** [review]
Looks great! Thanks Maggie!

**Review by KevinFCormier — APPROVED** [review]

**Pull request merged**